lib/resources/odata.js

```diff
@@ -37,7 +37,7 @@ module.exports = (service, { Form, Submission, env }) => {
     Form.getByXmlFormId(params[0]) // first regexp match
       .then(getOrNotFound)
       .then((form) => auth.canOrReject('read', form)
-        .then(() => Submission.getById(form.id, params[1])
+        .then(() => Submission.getById(form.id, params[1], true)
```

Comment:
The `true` here ensures that the join on the actor is always performed so the name can be pulled but it's not possible to get the full actor object in the odata feed, right?

Reply:
Correct. The way these records get pulled apart nothing is ever automatically passthrough.

Reply:
And besides, actor information is really really boring. It's _user_ information that contains interesting things, like emails and password hashes. Actors are pretty much just id and displayname (and timestamp metadata).

Reply:
That makes sense. I was getting a little turned around regarding the meaning of the extended parameter vs. the extended header.

Reply:
often the latter triggers the former!